configure.ac

```diff
@@ -1494,7 +1491,10 @@ AC_SUBST(UNIVERSAL_ARCH_FLAGS)
 # tweak BASECFLAGS based on compiler and platform
 case $GCC in
 yes)
-    CFLAGS_NODIST="$CFLAGS_NODIST -std=c99"
+    if test "$CC" != 'g++' ; then
```

Comment:
I have doubts that this test makes sense. First, g++ is not used for compiling the Python interpreter, because C99 constructions incompatible with C++ are used in CPython sources. Second, it doesn't work for e.g. `CC=/usr/bin/x86_64-linux-gnu-g++-7`. It would be more reliable to use a test similar to tests for options `-Wunused-result` etc.

Reply:
Added check for -Wstrict-prototypes